src/beautifiers/goimports.coffee

```diff
@@ -0,0 +1,20 @@
+###
+Requires https://godoc.org/golang.org/x/tools/cmd/goimports
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Goimports extends Beautifier
+  name: "goimports"
+  link: "https://godoc.org/golang.org/x/tools/cmd/goimports"
+  isPreInstalled: false
```

Comment:
This has been deprecated.

We have switched to using `Executables`: https://github.com/Glavin001/atom-beautify/issues/1687

See https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/sass-convert.coffee#L7-L20 and https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/sass-convert.coffee#L30 for simple example. And another: example https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/autopep8.coffee#L12-L56

Please let me know if you have any questions! This is a new process to help standardize creating non-preinstalled beautifiers. Thanks!